### ISSUE TITLE
Rule `no-async-fn-without-await` doesn't report when test has a title

### ISSUE THREAD

**Issue by sindresorhus** [avajs/eslint-plugin-ava#160]
The following reports:

```js
test(async t => {});
```

But this does not:

```js
test('foo', async t => {});
```

---

We should audit our tests for all our rules to ensure they all have a test for with and without a title.

---

// @jfmengels
